src/mock_data/storybook_mock_data.tsx

```diff
@@ -1,10 +1,38 @@
 import { LessonButtonCardProps } from '../components/LessonButtonCard/LessonButtonCard';
 import { LessonListHeaderProps } from '../components/LessonListHeader/LessonListHeader';
 import { ChapterButtonCardProps } from '../components/ChapterButtonCard/ChapterButtonCard';
+import { PDFEmbedProps } from '@/components/PDFEmbed/PDFEmbed';
+import { VideoPlayerProps } from '@/components/VideoPlayer/VideoPlayer';
+import { VideoHeaderProps } from '@/components/VideoHeader/VideoHeader';
+import { PDFDownloadButtonProps } from '@/components/PDFDownloadButton/PDFDownloadButton';
 
 // surah name mock data
 export const sampleSurahName = 'Aal-E-Imran';
 
+export const sampleSrcPDF: PDFEmbedProps = {
+  src: 'https://www.alislam.org/quran/Holy-Quran-English.pdf',
```

Comment:
Once we get the s3 stuff figured out, let's switch this out for a qj note, otherwise lgtm